Sources/SystemMetrics/SystemMetrics.swift

```diff
@@ -121,17 +121,17 @@ extension MetricsSystem {
     }
 }
 
-public enum SystemMetrics {
+public enum SystemMetrics: Sendable {
     /// Provider used by `SystemMetrics` to get the requested `SystemMetrics.Data`.
     ///
     /// Defaults are currently only provided for linux. (`SystemMetrics.linuxSystemMetrics`)
-    public typealias DataProvider = () -> SystemMetrics.Data?
+    @preconcurrency public typealias DataProvider = @Sendable () -> SystemMetrics.Data?
```

Comment:
@FranzBusch because those functions are passed as this type, which needs to be Sendable for some reason I can’t recall but I was just doing compiler driven annotations until it built.

Reply:
Maybe because the enum itself is Sendable and they are static members of the enum?